### ISSUE TITLE
Keeps jumping back to previous location 

### ISSUE THREAD

**Issue by newnumber2** [bryceco/GoMap#412]
I am very annoyed by this new bug. I open the app, touch the GPS tool, turn tool off, do my editing, and upload. As soon as the upload is finished, it jumps me back to the place I had been editing the last time I used the app.

**Comment by bryceco** [comment]
You're the second person to report this, but I haven't seen it and as far as I'm aware none of the beta testers have either. I'm really at a loss as to the cause, or how to fix it. Does it happen if you kill the app and then restart it without enabling gps?

**Comment by bryceco** [comment]
Also, if you're not part of the TestFlight beta please join it and try the beta there.

**Comment by newnumber2** [comment]
Yes. I only did one edit today (new restaurant) and after I added itand saved it, it jumped back to the location I had last mapped at(probably over a week ago). And yes, it starts at the last mapped areawhen you open the app, which is fine, of course. But I haven't triedediting at a moment like that. 

Be seeing you,

David J. Wisbey

(720) 422-0289

 

    On Monday, August 31, 2020, 9:54:51 PM MDT, bryceco <notifications@github.com> wrote:  
 
 


You're the second person to report this, but I haven't seen it and as far as I'm aware none of the beta testers have either. I'm really at a loss as to the cause, or how to fix it. Does it happen if you kill the app and then restart it without enabling gps?
